docs/introduction/faq.md

```diff
@@ -36,26 +36,48 @@ a vehicle in which to make WebVR successful.
 
 ## How can I get started?
 
-[contact]: #how-can-i-get-in-touch-with-the-aframe-team
-[homepage]: https://aframe.io
+[intro]: ../introduction/index.md
+[gettingstarted]: ../introduction/getting-started.md
 [guides]: ../guides/index.md
 
-A-Frame is very extensive across different disciplines so it can be difficult
-to know how to get started. We've tried our best to compile resources to get
-your started on your journey.
+Read the [Introduction][intro] to get a deeper feel of what A-Frame is.
 
-The examples on the [homepage][homepage] act as a starting point for you to
-read, copy, and paste code to quickly get started. Each example comes with an
-accompanying guide explaining how it was built and what components were used to
-built it that will help your understanding.
+See the [Getting Started][gettingstarted] guide to get set up.
 
-Below the examples are various workflow [guides][guides] featuring different
-tools and libraries. Select a guide based on what you are comfortable with
-(e.g., whether you know how to code, whether you know a certain framework,
-whether you know how to model).
+Check out the [guides][guides] for basic tutorials.
 
-If you need additional support, [contact us][contact]! We are very interested
-in understanding learning curve difficulties.
+Past that, you can browse the resources on the [`awesome-aframe`
+repository][awesome]. If you need additional support, [ask on
+StackOverflow][stackoverflow].
+
+## How is A-Frame's performance?
+
+Historically, JavaScript and the browser have a bad reputation for performance
+compared to native. This is perhaps due to DOM and layout, but in the case of
+WebVR, we are able to run 90+ FPS experiences in the HTC Vive in the browser.
+See [A-Painter](https://blog.mozvr.com/a-painter/) for an example of a full
+application. WebGL is quite capable.
+
+A-Frame itself should not introduce much overhead on top of [three.js][three]
+despite being built on top of Custom Elements. We've modified the HTML element
+prototypes to remove some performance limitations, such that A-Frame almost has
+its own flavor of virtual DOM:
+
+- The effects of `setAttribute` are synchronous. When modifying an entity's
+position, there is almost zero overhead. Doing `setAttribute('position', {x:
+1, y: 2, z: 3})` is practically directly modifying the underlying three.js
+objects themselves.
+- When changing an entity's properties, data is not serialized back to the DOM. This
+reduces the DOM overhead and most modifications are done directly in memory,
+similar to Virtual DOM.
+- HTML attributes explicitly set from an HTML file are cached in memory.
+
+[bestpractices]: ../introduction/best-pratices.md
+
+However, the performance of complex A-Frame scenes depends on the optimizations
+that developers employ. In 3D and game development, resources are more scarce,
```

Comment:
`Resource are more scarce` sounds like a strange statement.  What about something like this:

`In a-frame we provide a set of reasonable defaults that yield good performance for the most common use cases. However, performance is ultimately determined by the complexity and characteristics of each individual scene. In order to get the best use of resources developers will need understanding on how a 3D graphics pipeline works. These are some guidelines that will help you get started`